### DISCUSSION TITLE
[Projects Beta] Internal visibility

### DISCUSSION THREAD

**Discussion by dserodio** [community/community#8230] [Enterprise]
Currently, a project can be either Private or Public. I'd like to have an "Internal" visibility, so every use of my organization is able to interact with this project. I'm using GitHub Enterprise Cloud.

![image](https://user-images.githubusercontent.com/233589/143952380-c362df85-4494-4336-a6b4-dac8cc48abf7.png)

**Comment by joshuaNathaniel** [comment]
Workaround is to create an org group and add it as a collaborator.

**Reply by betiana-felice-snkeos** [reply]
Is it part of the roadmap to release the internal visibility feature soon?

**Comment by Timmmy-nlb** [comment]
We have the same requirement as enterprise cloud users (with many orgs).
And it's a real pity that this issue has been lying around for so long.
